_assets/stylesheets/modules/_syntax.scss

```diff
@@ -0,0 +1,218 @@
+// =============================================================================
+// Syntax
+// =============================================================================
+
+.highlight {
+  display: block;
```

Comment:
Properties should be ordered background, border-radius, color, display, font-weight, margin, padding